src/components/Api.js

```diff
@@ -0,0 +1,25 @@
+export default class Api {
+  constructor({ url, method, headers, body }) {
+    this._url = url;
+    this._method = method || "GET";
+    this._headers = headers;
+    this._body = body;
+  }
+
```

Comment:
In `Api` there should be all methods for fetching data: `getInitialCards`, `geUserInfo` and so on and only then they should be called in `index.js`